src/Components/Login/Login.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import './Login.css'
+import { useQuery, gql } from '@apollo/client'
+
+interface User {
+  userName: string,
+  emailAddress: string,
+  location: string
+}
+
+const ALL_USERS = gql `
+  query AllUsers {
+    users {
+      userName
+      emailAddress
+      location
+    }
+  }
+`
+
+const ALL_BOOKS = gql `
+  query AllBooks {
+    books {
+      id
+      imageUrl
+    }
+  }
+`
+
+const Login = () => {
+  const [email, setEmail] = useState('')
+  const [username, setUsername] = useState('')
+  const [userLocation, setUserLocation] = useState('')
+  const [activeAccount, setAccount] = useState(true)
+  const [allUsers, setUsers] = useState([{userName: "Josh", emailAddress: "test", location: "Denver"}])
```

Comment:
We talked in person, but we as a front end don't need to save this in state, we just need to hit the back end with a user name and they will send a 200 response if its a username in the database or not if the user name doesn't exist in the database